eq-author/src/App/questionPage/Preview/index.test.js

```diff
@@ -70,6 +77,55 @@ describe("PreviewPageRoute", () => {
     const wrapper = shallow(
       <PreviewPageRoute loading={false} data={{ questionPage }} />
     );
-    expect(wrapper.exists('[data-test="definition"]')).toBeFalsy();
+    expect(wrapper.exists(byTestAttr("definition"))).toBeFalsy();
+  });
+
+  it("should render definition label missing message", () => {
```

Comment:
Should we be rendering the definition label if the message is missing? Runner seems to drop the whole thing if there's no message so the preview would be incorrect, not sure how to make it clear that the reason it's not displayed is due to the missing label though?

Reply:
@hamishtaplin what are your thoughts on this?